pkg/neighbor/cmd.go

```diff
@@ -6,7 +6,7 @@ import "strings"
 // external
 // internal
 
-// SetTestCmd sets the test command that will be run on external projects.
-func SetTestCmd(c *Ctx) {
-	c.TestCmd = strings.Split(c.Config.Contents.TestCmdStr, " ")
+// SetExternalCmd sets the command that will be run on external projects.
+func SetExternalCmd(c *Ctx) {
+	c.ExternalCmd = strings.Split(c.Config.Contents.ExternalCmdStr, " ")
```

Comment:
This won't handle calls that contain arguments denoted with `"`. We should note this in documentation or support it.

Reply:
it should. I can confirm. It should just split on an empty space. I don't think that the quotes are part of the split string